rekolekcje-engine/src/test/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantsTest.java

```diff
@@ -10,7 +10,7 @@
 
 public abstract class ParticipantsTest {
 
-  protected ParticipantsService service = new ParticipantsConfiguration().participantsService();
+  protected ParticipantsFacade service = new ParticipantsConfiguration().participantsService();
```

Comment:
Wygląda na to, że w `ParticipantConfiguration` nie zmieniłeś nazwy na `*Facade`

Reply:
Racja! Dobrze, że to wyłapałeś :)